tool/tsh/tsh.go

```diff
@@ -1086,43 +1090,69 @@ func showApps(servers []services.Server, verbose bool) {
 	}
 }
 
-func showDatabases(servers []types.DatabaseServer, active []tlsca.RouteToDatabase, verbose bool) {
+func showDatabases(cluster string, servers []types.DatabaseServer, active []tlsca.RouteToDatabase, verbose bool) {
 	if verbose {
-		t := asciitable.MakeTable([]string{"Name", "Description", "URI", "Labels"})
+		t := asciitable.MakeTable([]string{"Name", "Description", "Protocol", "URI", "Labels", "Connect"})
 		for _, server := range servers {
 			name := server.GetName()
+			var connect string
 			for _, a := range active {
 				if a.ServiceName == name {
 					name = formatActiveDB(a)
+					connect = formatConnectCommand(cluster, a)
 				}
 			}
 			t.AddRow([]string{
 				name,
 				server.GetDescription(),
+				server.GetProtocol(),
 				server.GetURI(),
 				server.LabelsString(),
+				connect,
 			})
 		}
 		fmt.Println(t.AsBuffer().String())
 	} else {
-		t := asciitable.MakeTable([]string{"Name", "Description", "Labels"})
+		t := asciitable.MakeTable([]string{"Name", "Description", "Labels", "Connect"})
 		for _, server := range servers {
 			name := server.GetName()
+			var connect string
 			for _, a := range active {
 				if a.ServiceName == name {
 					name = formatActiveDB(a)
+					connect = formatConnectCommand(cluster, a)
 				}
 			}
 			t.AddRow([]string{
 				name,
 				server.GetDescription(),
 				server.LabelsString(),
+				connect,
 			})
 		}
 		fmt.Println(t.AsBuffer().String())
 	}
 }
 
+// formatConnectCommand formats an appropriate database connection command
+// for a user based on the provided database parameters.
+func formatConnectCommand(cluster string, active tlsca.RouteToDatabase) string {
+	switch active.Protocol {
+	case defaults.ProtocolPostgres:
+		service := fmt.Sprintf("%v-%v", cluster, active.ServiceName)
+		switch {
+		case active.Username != "" && active.Database != "":
+			return fmt.Sprintf(`psql "service=%v"`, service)
+		case active.Username != "":
+			return fmt.Sprintf(`psql "service=%v dbname="`, service)
```

Comment:
Can you clarify the meaning of attributes w/o a value (e.g. `dbname=`)? Are these setting an empty value?

Reply:
It was supposed to indicate that user needs to provide their own values, I agree it was confusing. Updated.